arduino-ide-extension/src/browser/boards/boards-service-provider.ts

```diff
@@ -270,7 +270,6 @@ export class BoardsServiceProvider implements FrontendApplicationContribution {
   }
 
   protected setBoardsConfig(config: BoardsConfig.Config): void {
-    this.logger.info('Board config changed: ', JSON.stringify(config));
```

Comment:
Why did you remove it?

Reply:
I still think it's kinda noisy during development, but you find it useful I'll re-add it, no prob 👍